hadrons/functions.py

```diff
@@ -137,7 +140,7 @@ def calc_b_cm(LET_keV_um):
     Calculate the Gaussian track radius as suggested by Rossomme et al.
     Returns the track radius in cm given a LET [keV/um]
     '''
-    data = np.genfromtxt("data_LET/LET_b.dat", delimiter=",", dtype=float)
+    data = np.genfromtxt(f"{ABS_PATH}/data_LET/LET_b.dat", delimiter=",", dtype=float)
```

Comment:
It seems that `np.genfromtxt` accepts Path object as well:
https://numpy.org/doc/stable/reference/generated/numpy.genfromtxt.html

```suggestion
    data = np.genfromtxt(Path(ABS_PATH, "data_LET", "LET_b.dat"), delimiter=",", dtype=float)
```